test/cmd.js

```diff
@@ -66,6 +67,7 @@ describe('express(1)', function () {
       assert.strictEqual(contents, '{\n' +
         '  "name": "express-1-no-args",\n' +
         '  "version": "0.0.0",\n' +
+        '  "type": "commonjs",\n' +
```

Comment:
Is this still required?

Reply:
As I quoted earlier, it's recommended in the Node docs to explicitly set type for every package. So, no, it's not required, but I agree with the thinking that it should be included.